gh-push/handler.go

```diff
@@ -92,8 +92,17 @@ func Handle(req []byte) string {
 		return msg
 	}
 
+	serviceValue := fmt.Sprintf("%s-%s", pushEvent.Repository.Owner.Login, pushEvent.Repository.Name)
+
+	eventInfo := sdk.BuildEventFromPushEvent(pushEvent)
+	status := sdk.BuildStatus(eventInfo, "")
+	status.AddStatus(sdk.Pending, fmt.Sprintf("%s stack deploy is in progress", serviceValue), sdk.Stack)
```

Comment:
The pending status looks good in the UI

Since the status is created for the login/repo - how do you update that later to show the success or failure overall?

Reply:
The context here is used : `sdk.Stack`.  The same context is being updated on success at [git-tar](https://github.com/s8sg/openfaas-cloud/blob/d2b7a8897b8f6718d122a1d1d580669350a22554/git-tar/function/handler.go#L71)
```go
status.AddStatus(sdk.Success, "stack is successfully deployed", sdk.Stack)
```

Reply:
Ok makes sense 👍